go/types/encode_human_readable.go

```diff
@@ -178,6 +239,14 @@ func (w *hrsWriter) writeStruct(v Struct, printStructName bool) {
 		w.write(" ")
 	}
 	w.write("{")
+	commenters := GetHRSCommenters(v.name)
+	for _, commentFunc := range commenters {
+		if comment := commentFunc(v); comment != "" {
+			w.write(" // " + comment)
```

Comment:
This seems to print `// a // b // c` if three commenters are registered that returns non empty strings. Not sure what we want to do here though?